app/packages/embeddings/src/useViewChangeEffect.tsx

```diff
@@ -47,6 +47,12 @@ export function useViewChangeEffect() {
     fetchPlot({ datasetName, brainKey, view, labelField })
       .catch((err) => setLoadingPlotError(err))
       .then((res) => {
+        if (!res || !res.index_size) {
```

Comment:
Shouldn't `res.index_size == 0` have a different warning? Maybe the user loaded `dataset.limit(0)` view. Something like:
```
The index contains no embeddings for the current view.
```

Reply:
`res.index_size == 0` is handled below. This catches if there is no `index_size` on the result object, which was the case I hit. Instead it contained an error message (didn't throw an error that would have been caught by the catch above)

Reply:
Index size === 0 is being caught here since we return and !0 === !undefined

Reply:
updated to return a different message when index is 0 vs no index at all